utils/text_extractor.py

```diff
@@ -24,15 +25,15 @@ def findTitle(html):
 
 
 def get_all_text_from_url(url, retries=5):
-    if retries == 0:
+    if retries < 1:
         logging.error("get_all_text_from_url: could not handle request")
         raise Exception("could not handle request")
     try:
```

Comment:
Unless I missed something please fix only things related to the JIRA ticket it might create a chaos

Reply:
ho sorry your right that makes sense
should I change it back or is it ok?

Reply:
yes and on the way you can practice interactive rebase!